include/prism/node.h

```diff
@@ -53,6 +57,7 @@ void pm_node_list_free(pm_node_list_t *list);
  *
  * @param parser The parser that owns the node.
  * @param node The node to deallocate.
+ * \public \memberof pm_node \relatedalso pm_parser
```

Comment:
I think this is the opposite.

```suggestion
 * \public \memberof pm_parser \relatedalso pm_node
```